pdata/plog/plogotlp/logs.go

```diff
@@ -74,6 +74,13 @@ func NewRequest() Request {
 	return Request{orig: &otlpcollectorlog.ExportLogsServiceRequest{}}
 }
 
+// NewRequestFromLogs returns a Request from plog.Logs.
+// Note that Request is essentially just a wrapper for plog.Logs,
```

Comment:
This line should be also in the "Request" struct comment. then this comment can be simplified like:

// Because Request is a wrapper for plog.Logs, any changes to the provided Logs struct will be reflected in the Request and vise versa.

Reply:
Updated